client/post-editor/post-editor.jsx

```diff
@@ -807,6 +812,7 @@ export default connect(
 			postId,
 			showDrafts: isEditorDraftsVisible( state ),
 			editorModePreference: getPreference( state, 'editor-mode' ),
+			editorSidebarPreference: getPreference( state, 'editor-sidebar' ),
```

Comment:
If this is `null` should we default to `open` so the sidebar is open by default?